src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportAccountsToPL.sql

```diff
@@ -0,0 +1,173 @@
+﻿CREATE PROCEDURE [dbo].[ImportAccountsToPL]
```

Comment:
can we split this SP in to 2 ImportAccountsToPL and ImportAcc_AccountLegalEntityToPL so that we can benefit from having management logs separately. If we leave it this way I believe the second import won't take place if the first one fails for any reason?